src/main/java/de/sakpaas/backend/service/LocationDetailsRepository.java

```diff
@@ -1,11 +1,16 @@
 package de.sakpaas.backend.service;
 
 import de.sakpaas.backend.model.LocationDetails;
+import java.util.List;
 import java.util.Optional;
 import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
 
 public interface LocationDetailsRepository extends JpaRepository<LocationDetails, Long> {
 
   Optional<LocationDetails> findById(Long id);
 
+  @Query(value = "SELECT DISTINCT type FROM location_details", nativeQuery = true)
```

Comment:
Ist das Performanter?

Und haben wir da einen `index` drauf? Sonst ist das immer ein full Table scan

Reply:
Schau mal in den Kommentar über dir :)

Reply:
Ne ist gleich Perfomant, Fabio fand das nur schöner. Aktuell haben wir noch keinen Index drauf soweit ich weiß, braucht aber trotzdem nur 50ms.